### ISSUE TITLE
etc/rrdcached-default-lsb.in missing from 1.7.1 tarball

### ISSUE THREAD

**Issue by atsampson** [oetiker/rrdtool-1.x#956]
`etc/rrdcached-default-lsb.in` isn't included in `rrdtool-1.7.1.tar.gz`. This causes `configure` to fail when it tries to expand the file:

```
config.status: error: cannot find input file: `etc/rrdcached-default-lsb.in'
```

I guess this needs adding to `etc/Makefile.am` so `make dist` knows about it (are there other files in `etc` that should also be listed there?).

**Comment by oetiker** [comment]
oops ... thanks ... created new tarball

**Comment by oetiker** [comment]
at the moment etc content is included via the toplevel Makefile.am ... but you are right this could be migrated don into this one.

**Comment by c72578** [comment]
What about the following files, which were part of rrdtool-1.7.0.tar.gz and are not in rrdtool-1.7.1.tar.gz:
`etc/rrdcached.service.in`
`etc/rrdcached.socket.in`

**Comment by oetiker** [comment]
now it is properly fixed :) and the tarball recreated again

**Comment by atsampson** [comment]
That configures and builds successfully for me - thanks!

Running the test suite reveals that `tests/graph1.output` is missing too, though...

**Comment by oetiker** [comment]
we are getting there ... slowly :)

**Comment by c72578** [comment]
What about `etc/rrdcached-init-lsb`, should it also go into the tarball?
If yes, I would add it to EXTRA_DIST in `etc/Makefile.am` and also remove the trailing space after the backslash in `etc/Makefile.am:1`

**Comment by oetiker** [comment]
yes please

**Comment by c72578** [comment]
Fixed in v1.7.2
